src/page-content/navigation/header.json

```
@@ -39,7 +39,7 @@
           "to": "/hydra"
         },
         {
-          "title": "Global Authorization Server",
+          "title": "Global Authorization",
```

```suggestion
          "title": "Global Authorization Service",
```

What do you think of this? was patricks suggestion.

"Global Authorization" is a bit vague, that could mean anything from KYC to bodyguard services.